packages/atomic/src/components/common/interface/interface-common.tsx

```diff
@@ -31,6 +31,12 @@ export interface BaseAtomicInterface<EngineType extends AnyEngineType>
   registerFieldsToInclude(): void;
 }
 
+export const mismatchedInterfaceAndEnginePropError = (
+  interfaceKind: 'search' | 'recommendation',
+  configurationName: 'query pipeline' | 'search hub'
+) =>
+  `A ${configurationName} is configured on the ${interfaceKind} interface, but the ${interfaceKind} interface was initialized with an engine. You should only configure the ${configurationName} in the engine.`;
```

Comment:
@jpmarceau 

this is another string that needs to be validated

Reply:
```suggestion
  `A ${configurationName} is configured on the ${interfaceKind} interface element, but the ${interfaceKind} interface was initialized with an engine. You should only configure the ${configurationName} in the target engine.`;
```